addon/modifiers/sortable-item.js

```diff
@@ -0,0 +1,762 @@
+import Ember from 'ember';
```

Comment:
Just one minor thing, can we add the change from #347 into the modifier as well?
It'll keep both component and modifier in sync when I merge this.

Reply:
Yea Ill get to that tonight.

Should in future extract some of those methods to a util that could be shared for better maintenance

Reply:
Agreed, once this change is in, any latter improvements should be easily streamline-able. 
After this, my plan is to release a `beta` for anyone to try out the `modifier` and to give us ample time to get and fix bug reports. Eventually, we should keep the `modifier` and kill off the `component` in a `v3` release.